refinery_cli/src/migrate.rs

```diff
@@ -36,16 +36,20 @@ fn run_migrations(
     let mut migrations = Vec::new();
     for path in migration_files_path {
         let sql = std::fs::read_to_string(path.as_path())
-            .with_context(|| format!("could not read migration file name {}", path.display()))?;
+            .with_context(|| format!("Could not read contents of file {}", path.display()))?;
 
         //safe to call unwrap as find_migration_filenames returns canonical paths
         let filename = path
             .file_name()
             .and_then(|file| file.to_os_string().into_string().ok())
             .unwrap();
 
-        let migration = Migration::unapplied(&filename, &sql)
-            .with_context(|| format!("could not read migration file name {}", path.display()))?;
+        let migration = Migration::unapplied(&filename, &sql).with_context(|| {
+            format!(
+                "Could not create new unapplied migration with filename {}",
```

Comment:
do we want to mention to the user that we are creating an unapplied `Migration`? I'd say it can be a little confusing as it's giving internal information while the user should only care if the file could or could not be read wdyt?

Reply:
Yes you are right, I changed the message.